### DISCUSSION TITLE
JSBSim and s-function have different results and different reactivity themselves

### DISCUSSION THREAD

**Discussion by DDDDDahye** [JSBSim-Team/jsbsim#949] [Support requests]
In order to configure the controller of the model with a laddervator, JSBSim was called to Matlab/Simulink using S-function.
During this process, the JSBSim internal module (FGTurbin.cpp) was modified and a new library was created and loaded.

Two S-function references were found and applied to both.

![image](https://github.com/JSBSim-Team/jsbsim/assets/38837057/fd07fdf7-01ca-4b88-ace8-c29c90ea10bd)

![image](https://github.com/JSBSim-Team/jsbsim/assets/38837057/ca7b90d6-7d70-4d9b-8426-25dc5ba50896)

However, both versions of JSBSim v1.13 and earlier had completely different values of JSBSim and s-function and different reactivity themselves.

![image](https://github.com/JSBSim-Team/jsbsim/assets/38837057/4c6b5d07-e6dc-41f6-af1d-aacac3ef1b89)

![image](https://github.com/JSBSim-Team/jsbsim/assets/38837057/c5e0576e-8f9e-4759-aa7a-f2ff13adb6f5)

It was confirmed that the control surface command and the control surface position were the same.

![image](https://github.com/JSBSim-Team/jsbsim/assets/38837057/9d9a4dab-5c87-45e8-aefc-815d195123c3)

![image](https://github.com/JSBSim-Team/jsbsim/assets/38837057/c3f1c4d5-bff5-4ff9-a9bc-3449701d6d33)

![image](https://github.com/JSBSim-Team/jsbsim/assets/38837057/a14380b7-fde6-4b8d-a35b-0e31a3576b30)

![image](https://github.com/JSBSim-Team/jsbsim/assets/38837057/b40244d3-a949-483b-9b53-4da4fe356511)

![image](https://github.com/JSBSim-Team/jsbsim/assets/38837057/7dd5faf7-2360-4c78-9cde-61c137998d44)

I've all used the same xml file, but I don't know when I get a different result from S-function.

The s-function shows a maneuver that seems physically impossible.

Does anyone know when this happens? What did I miss?

Thanks in advance!

**I'm submitting a ...**
  - [ ] bug report
  - [ ] feature request
  - [v] support request

**Comment by bcoconni** [comment]
This issue is a support request and as was explicitly mentioned in the issue template : "**This repository's issues are reserved for feature requests and bug reports. Issues that are neither bug reports or feature requests will be transferred to [Discussions](https://github.com/JSBSim-Team/jsbsim/discussions) by the team and the issue will be locked and closed.**"

This issue is therefore closed and converted to a discussion.
